subsystems/DiscordRPC/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    java
+    `java-library`
```

Comment:
Gradle confuses me 🧐  Do you use backticks, single quotes, or double quotes here in the plugin section?

Reply:
It is kotlin.
You can write methods with spaces too(useful for test names).

Reply:
💡